e2e-tests/tfm/cypress/e2e/journeys/deal-cancellation/submit-deal-cancellation-in-past.spec.js

```diff
@@ -36,45 +38,63 @@ context('Deal cancellation - effective from date in past', () => {
     });
   });
 
+  const submitDealCancellation = () => {
+    caseDealPage.cancelDealButton().click();
+
+    cy.url().should('eq', relative(`/case/${dealId}/cancellation/reason`));
+    cy.clickContinueButton();
+
+    cy.url().should('eq', relative(`/case/${dealId}/cancellation/bank-request-date`));
+    cy.completeDateFormFields({ idPrefix: 'bank-request-date' });
+    cy.clickContinueButton();
+
+    cy.url().should('eq', relative(`/case/${dealId}/cancellation/effective-from-date`));
+    cy.completeDateFormFields({ idPrefix: 'effective-from-date', date: yesterday.date });
+    cy.clickContinueButton();
+
+    cy.url().should('eq', relative(`/case/${dealId}/cancellation/check-details`));
+    checkDetailsPage.dealDeletionButton().click();
+  };
+
   describe('when logged in as a PIM user', () => {
     before(() => {
       cy.login(PIM_USER_1);
-
       cy.visit(relative(`/case/${dealId}/deal`));
+    });
 
-      caseDealPage.cancelDealButton().click();
+    describe('after submitting the cancellation', () => {
+      it('should redirect you to the deal summary page with the success banner visible once', () => {
```

Comment:
I have to do all this in one test - if it's split out into multiple tests then the window get's refreshed/reloaded as part of starting up the new test, which means the success banner disappears